tests/broadcast.rs

```diff
@@ -214,10 +216,57 @@ fn test_broadcast_different_sizes<A, F>(
     }
 }
 
-#[test]
-fn test_8_broadcast_equal_leaves_silent() {
+proptest! {
+    #![proptest_config(ProptestConfig {
+        cases: 1, .. ProptestConfig::default()
+    })]
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_8_broadcast_equal_leaves_silent(seed in gen_seed()) {
+        do_test_8_broadcast_equal_leaves_silent(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_random_delivery_silent(seed in gen_seed()) {
+        do_test_broadcast_random_delivery_silent(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_first_delivery_silent(seed in gen_seed()) {
+        do_test_broadcast_first_delivery_silent(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_first_delivery_adv_propose(seed in gen_seed()) {
+        do_test_broadcast_first_delivery_adv_propose(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_random_delivery_adv_propose(seed in gen_seed()) {
+        do_test_broadcast_random_delivery_adv_propose(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_random_delivery_adv_propose_and_drop(seed in gen_seed()) {
+        do_test_broadcast_random_delivery_adv_propose_and_drop(seed)
+    }
+
+    #[test]
+    #[allow(clippy::unnecessary_operation)]
+    fn test_broadcast_random_adversary(seed in gen_seed()) {
+        do_test_broadcast_random_adversary(seed)
+    }
+}
+
+fn do_test_8_broadcast_equal_leaves_silent(seed: TestRngSeed) {
     let new_adversary = || ReorderingAdversary::new();
-    let mut rng = rand::thread_rng();
+    let mut rng: TestRng = TestRng::from_seed(seed);
```

Comment:
There's another unnecessary closure above that line.

Reply:
Fixed